docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -1716,6 +1716,9 @@ See <<disk-based-shard-allocation>> for more information about disk watermarks a
 
 `io_stats` (Linux only)::
 (objects) Contains I/O statistics for the node.
+
+**NOTE**: Due to limitations of the Linux kernel, When running the application within a container such as Docker, these statistics may not be accurate.
```

Comment:
Suggest using `NOTE:` so this is formatted as a proper [admonition](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/).

```suggestion
NOTE: Due to limitations of the Linux kernel, When running the application within a container such as Docker, these statistics may not be accurate.
```